cloud-optimized-geotiffs/cogs-overview_resampling.ipynb

```diff
@@ -1,19 +1,20 @@
 {
```

Comment:
<p>No it's Types of Data - Nominal, Ordinal, Discrete, Continuous. Also the variance plays a role as well has how much NA data there is.</p><p> </p><p>Data Type to me implies, byte, integer, float, etc...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I changed it back and added types of data in parenthesis for clarity of context.